accounts/wallet_account.go

```diff
@@ -0,0 +1,40 @@
+package accounts
+
+import (
+	"fmt"
+)
+
+type errInvalidAccountAddress struct {
+	account Account
+}
+
+func (err errInvalidAccountAddress) Error() string {
+	return fmt.Sprintf("invalid account address, doesnt exists in wallet: %s" + err.account.Address.String())
+}
+
+func newErrInvalidAccountAddress(account Account) errInvalidAccountAddress {
+	return errInvalidAccountAddress{account}
+}
+
```

Comment:
noticed just know that the accounts package contains an errors.go file with all the errors.

Reply:
will move there